doc/api/deprecations.md

```diff
@@ -523,6 +523,12 @@ Type: Runtime
 of V8 5.8. It is replaced by Inspector which is activated with `--inspect`
 instead.
 
+<a id="DEP00XX"></a>
```

Comment:
Do these placeholders get replaced when making a release, like with the API version metadata or ?